### DISCUSSION TITLE
Proton Calendar Issues

### DISCUSSION THREAD

**Discussion by frecklesmcgee** [derekantrican/GAS-ICS-Sync#425] [Q&A]
Has anyone tried this script to pull from a Proton calendar? I've run the script a few times using different ics urls for my proton calendar but it doesn't seem to be updating (I'm not getting error on the google / script side). Perhaps a Proton issue? I'm at a loss. Thanks for any tips / tricks!

**Comment by Lonestarjeepin** [comment]
I'm not familiar with Proton, but the script should be able to handle any publicly accessible ics file. Have you tried putting the ics URL into icalendar.org to check and make sure it's giving a valid ics?

**Comment by jonas0b1011001** [comment]
Closing due to lack of feedback.

**Comment by girlinabillion** [comment]
Hi! Could we reopen this issue? The Proton Calendar URL that I got has a CacheKey and PassPhraseKey included at the end of the URL. Maybe this is why it won't work? I did make sure that the link is shareable to anyone with the link.
